common/templates/table.html

```diff
@@ -0,0 +1,20 @@
+<div class="outer-table">
+    <table class="table table-striped table-bordered table-hover">
+        <!-- sortable column header -->
+        <thead>
+            <tr>
+                <th nowrap ng-repeat="col in tableModel.columns">
+                    <span class="table-column-displayname" ng-click="sortby(col.name)">{{col.displayname}}</span>
```

Comment:
Should this be `ng-click=sortBy`? (I'm assuming it's referring to the `scope.sortBy` in your directive.